### ISSUE TITLE
Ring Keypad + Alarmo v1.9.0 = no talking voices

### ISSUE THREAD

**Issue by crlogic** [nielsfaber/alarmo#457]
### Alarmo Version

v1.9.0

### HA Version

2022.5.0

### Bug description

HASSIO v2022.5.0 using [Ring keypad](https://community.home-assistant.io/t/synchronize-ring-alarm-keypad-v2-with-alarmo/349461) + Alarmo v1.8.6 = talking voices when (dis)arming.

Upgraded to Alarmo v1.9.0 = no voices/lights

Deleted custom_component folder, WinSCP'd Alarmo v1.8.6 and reloaded core = voices work again.

### Relevant log output

```shell
I need to enable some form of debugging for this?
```

**Comment by nielsfaber** [comment]
I don't really know how to help, this issue seems more related to the blueprint that you are using than alarmo itself.
I would suggest to just try arming/disarming alarmo manually (e.g. via alarmo-card), see if all works as you would expect.
If so, the problem is more likely caused by the blueprint, and I would suggest to ask for help in the forum thread instead.

I cannot think of any recent changes in Alarmo which could affect (break) this blueprint.
You can enable debug logging for alarmo by adding this to the configuration.yaml of HA and restarting HA: 
```
logger:
  default: warning
  logs:
    custom_components.alarmo: debug
```

**Comment by nielsfaber** [comment]
It wasn't very clear from your original report that the issue was specific to countdown time indication, which I definitely did change. For next time: please be more specific when reporting a bug.
This should be fixed by updating to release v1.9.1.

**Comment by stockmopar** [comment]
Thanks @nielsfaber! I'll update and give it a whirl. Your work on this is greatly appreciated!!!!
